tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c

```diff
@@ -419,11 +469,15 @@ ZTEST(wdt_basic_test_suite, test_wdt)
 		m_testcase_index++;
 #endif
 	}
-	if (m_testcase_index == 3U) {
+        if (m_testcase_index == 3U) {
+		zassert_true(test_wdt_enable_wait_mode() == TC_PASS);
+		m_testcase_index++;
+        }
+	if (m_testcase_index == 4U) {
 		zassert_true(test_wdt_bad_window_max() == TC_PASS);
 		m_testcase_index++;
 	}
-	if (m_testcase_index > 3) {
+	if (m_testcase_index > 4) {
```

Comment:
wouldn't 4 only occur if PM is enabled?

Reply:
I find some other issues with this test case, will update